Packs/Armorblox/Integrations/Armorblox/test_data/test_get_incident_by_id.json

```diff
@@ -0,0 +1,648 @@
+{
+    "incident_id": "12846",
+    "date": "2022-10-17T12:08:11Z",
+    "policy_names": [
+        "Abuse Report"
+    ],
+    "remediation_actions": [
+        "ALERT"
+    ],
+    "users": [
+        {
+            "user_name": "Admin",
+            "user_address": "admin@armorblox.dev",
```

Comment:
Replace all occurrences of this with a fake one - please check what we used last time.